.ci/test.sh

```diff
@@ -5,6 +5,8 @@ set -e
 
 if [[ "$TASK" == "rpkg" ]]; then
   R_PACKAGE_DIR=$(pwd)/r-pkg
+  Rscript .ci/lint.R ${R_PACKAGE_DIR}
```

Comment:
can you please lint the entire repo?

```
Rscript .ci/lint.R $(pwd)
```

As I mentioned, I think this linting code should hold our opinion about ALL R code in the project, not only what is in `r-pkg/`.